crates/starknet-server/tests/get_transaction_by_hash.rs

```diff
@@ -0,0 +1,102 @@
+pub mod common;
+
+mod get_transaction_by_hash_integration_tests {
+    use std::sync::Arc;
+
+    use starknet_core::constants::{DECLARE_V1_TRANSACTION_HASH, DECLARE_V2_TRANSACTION_HASH};
+    use starknet_rs_accounts::{Account, SingleOwnerAccount};
+    use starknet_rs_core::chain_id;
+    use starknet_rs_core::types::contract::{CompiledClass, SierraClass};
+    use starknet_rs_core::types::{
+        BlockId, BlockTag, BroadcastedDeclareTransactionV1, FieldElement,
+    };
+    use starknet_rs_providers::Provider;
+    use starknet_rs_signers::{LocalWallet, SigningKey};
+    use starknet_types::felt::Felt;
+    use starknet_types::traits::ToHexString;
+
+    use crate::common::constants::{
+        CASM_COMPILED_CLASS_HASH, PREDEPLOYED_ACCOUNT_ADDRESS, PREDEPLOYED_ACCOUNT_PRIVATE_KEY,
+    };
+    use crate::common::util::BackgroundDevnet;
+
+    #[tokio::test]
+    async fn get_declere_v1_transaction_by_hash_happy_path() {
```

Comment:
typo: declere

Reply:
Fixed in https://github.com/0xSpaceShard/starknet-devnet-rs/commit/04aa105def54f79b84f248b665c78928e62d14ab